### PR TITLE
Style/fix disabled linting rules

### PR DESCRIPTION
**Pull request by areisle**
I removed the disabling  and ran autofix for one rule per commit. there was only 1 where I then made a second commit to fix the couple spots manually that it couldn't autofix for that rule (which only involved moving code comments)

### CODE REVIEW

File: src/views/DataView/index.tsx [hunk 1]
```diff
@@ -313,7 +313,8 @@ const DataView = ({
     <div className={
         `data-view ${detailPanelIsOpen
           ? 'data-view--squished'
-          : ''}`}
+          : ''}`
+}
```
Comment (creisle) [inline src/views/DataView/index.tsx:317]: looks like something might be off with the spacing of the autofix here

File: src/views/GraphView/components/GraphComponent/GraphOptionsPanel/GraphOptionsPanel.tsx [hunk 2]
```diff
@@ -216,7 +214,8 @@ function GraphOptionsPanel(props) {
                     checked={
                         graphOptions.linksLegend
                         && graphOptions.linksColor
-                        && !linkLegendDisabled}
+                        && !linkLegendDisabled
+}
```
Comment (creisle) [inline src/views/GraphView/components/GraphComponent/GraphOptionsPanel/GraphOptionsPanel.tsx:218]: same autofix weirdness here

File: src/views/GraphView/index.tsx [hunk 3]
```diff
@@ -107,7 +107,8 @@ const GraphView = ({ history }) => {
     <div className={
       `data-view ${detailPanelIsOpen
         ? 'data-view--squished'
-        : ''}`}
+        : ''}`
+}
```
Comment (creisle) [inline src/views/GraphView/index.tsx:111]: more autofix werd spacing

File: src/static/graphkb-env-config.js [hunk 4]
```diff
@@ -9,4 +7,5 @@ window._env_ = {
   CONTACT_EMAIL: 'graphkb@bcgsc.ca',
   CONTACT_TICKET_URL: 'https://www.bcgsc.ca/jira/projects/KBDEV',
   PUBLIC_PATH: '/',
+  IS_DEMO: '',
```
Comment (creisle) [inline src/static/graphkb-env-config.js:10]: accidental commit?

**Review by creisle — CHANGES_REQUESTED** [review]

File: src/static/graphkb-env-config.js (hunk 4 above)
Comment (areisle) [reply src/static/graphkb-env-config.js:10]: yes, shoot. this file gets created every time you run the app and I'm constants having to revert it

File: src/views/GraphView/index.tsx (hunk 3 above)
Comment (areisle) [reply src/views/GraphView/index.tsx]: fixed in [edb87f9](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/edb87f9b15f38bb438ce748a51586105f00b3773)

File: src/views/GraphView/components/GraphComponent/GraphOptionsPanel/GraphOptionsPanel.tsx (hunk 2 above)
Comment (areisle) [reply src/views/GraphView/components/GraphComponent/GraphOptionsPanel/GraphOptionsPanel.tsx]: fixed in [3ae605b](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/3ae605b4002a665a6ea1e848c37f3bb098e6fec5)

File: src/views/DataView/index.tsx (hunk 1 above)
Comment (areisle) [reply src/views/DataView/index.tsx]: fixed in [3ae605b](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/3ae605b4002a665a6ea1e848c37f3bb098e6fec5)

File: src/static/graphkb-env-config.js (hunk 4 above)
Comment (areisle) [reply src/static/graphkb-env-config.js]: fixed in [edb87f9](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/edb87f9b15f38bb438ce748a51586105f00b3773)

**Review by creisle — APPROVED** [review]

**Pull request merged**